### DISCUSSION TITLE
How to load specified workflow other than the default graph at comfyui startup?

### DISCUSSION THREAD

**Discussion by Maslino** [Comfy-Org/ComfyUI#3282] [Q&A]
At comfyui startup, the default graph is loaded. How can I change the default graph? Is there anyway I can specify the workflow json file to be auto loaded at startup?

**Comment by ltdrdata** [comment]
ComfyUI-Custom-Scripts extension provides such feature.
https://github.com/pythongosssss/ComfyUI-Custom-Scripts

https://www.youtube.com/watch?v=QoLv5wLFkVg


Just so you know, this is a way to replace the default workflow, and basically, the workflow that pops up at startup is the final workflow cached at that URL. If you tunnel using something like Colab, the URL changes every time, so various features based on browser caching may not work properly.

**Reply by Maslino** [reply]
Thanks. I'll look into it.

**Reply by mieucun123** [reply]
> ComfyUI-Custom-Scripts extension provides such feature. https://github.com/pythongosssss/ComfyUI-Custom-Scripts
> 
> https://www.youtube.com/watch?v=QoLv5wLFkVg
> 
> Just so you know, this is a way to replace the default workflow, and basically, the workflow that pops up at startup is the final workflow cached at that URL. If you tunnel using something like Colab, the URL changes every time, so various features based on browser caching may not work properly.

it not working

**Reply by ltdrdata** [reply]
That is a very old way.
New front manages workflow natively.

**Reply by Shituation** [reply]
still there is no way to set the default workflow in 2025 (almost 2026)

**Comment by VideoFX** [comment]
I'm also looking for a way to edit or clear the default workflow.

**Reply by Shituation** [reply]
avoid this repo, its not the official. I fell for it too.
